Source/IFCExporterUIOverride/Properties/Resources.resx

```diff
@@ -789,4 +789,8 @@
   <data name="UseOnlyTriangulation" xml:space="preserve">
     <value>Keep Tessellated Geometry as Triangulation</value>
   </data>
+  <data name="DoorBelongsToRoom" xml:space="preserve">
+    <value>The door (IfcDoor) belongs to the room (IfcSpace)</value>
+    <comment>Used by eTASK Immobilien Software GmbH</comment>

```

Comment:
Please remove the comment - presumably this is a generally useful addition that anyone can use.  If it is really only for one piece of software, then I guess we shouldn't add it, but it seems like a good addition to me.

Reply:
Comment removed